lib/src/layer/tile_layer/tile_image_manager.dart

```diff
@@ -13,7 +13,7 @@ typedef TileCreator = TileImage Function(TileCoordinates coordinates);
 
 @immutable
 class TileImageManager {
-  final Map<String, TileImage> _tiles = {};
+  final Map<int, TileImage> _tiles = {};
```

Comment:
There is now a non-zero chance of hash collision.  Maybe the low probability is worth the performance?

Reply:
Maybe just key by the coordinate directly? and add a equals method.  With the cached hashcode it should avoid redundant calculations, and ensure collisions aren't a problem.

Reply:
That makes a lot of sense. Thanks.

I first wanted to encode the limited length x, y, z in the key, thus avoiding collisions but then switched back to a hash because I messed up integer precision in Javascript. In my defense it was late at night :hide:

PTAL